data/json/recipes/recipe_deconstruction.json

```
@@ -2147,6 +2147,13 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [ [ [ "scrap", 3 ] ], [ [ "clockworks", 2 ] ] ]
   },
+  {
+    "result": "bathroom_scale",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "6 m",
+    "components": [ [ [ "sheet_metal_small", 22 ] ], [ [ "clockworks", 3 ] ], [ [ "scrap", 3 ] ] ]
```

```suggestion
    "components": [ [ [ "sheet_metal_small", 4 ] ], [ [ "clockworks", 3 ] ], [ [ "scrap", 1 ] ] ]
```

??? Suddenly you're generating 5kg+ of steel here.